src/tools/dev_server/dev-server.ts

```diff
@@ -25,13 +26,15 @@ import {ExplorerProxy} from './explorer-proxy';
 
 const options = minimist(process.argv.slice(2), {
   boolean: ['verbose'],
-  default: {port: 8786, explorePort: 8787, verbose: false}
+  default: {port: 8786, explorePort: 8787, hotReloadPort: 8888, verbose: false}
 });
 
 const port = Number(options['port']);
 const explorePort = Number(options['explorePort']);
+const hotReloadPort = Number(options['hotReloadPort']);
 
 const proxy = new ExplorerProxy();
+const hotReloadServer = new HotReloadServer(hotReloadPort);
```

Comment:
It feels a little strange to me that we start a separate server from within this parent server. It seems like it's more that it's another chunk of logic, on a different port, only intended to start/run within the containing ALDS server.

Typically one would break something out into a separate server, and name it as a separate server, if you might want to run it individually/separately.

Mostly this is just commenting while reading, but if we don't expect/need to run it individually, maybe later we should consider a rename to make it clearer that it is really just a separate port handler / task / etc.